### ISSUE TITLE
[Bug]: DetailsRow and aria-busy

### ISSUE THREAD

**Issue by bhawnakumarigupta** [microsoft/fluentui#27645]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
Environment Information
Package version(s): Fluent UI React 8.106.2
Browser and OS versions: Microsoft Edge Browser Windows 11
```

### Are you reporting Accessibility issue?

yes

### Reproduction

NA

## Actual Behavior
Getting below accessibility error on `DetailsRow` Fluent UI component

Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (#row58-0)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Element path: #row58-0

Snippet:-
`<div data-is-focusable="true" id="row58-0" role="row" class="ms-FocusZone css-295 ms-DetailsRow root-469" data-selection-index="0" data-selection-touch-invoke="true" data-item-index="0" aria-rowindex="2" data-automationid="DetailsRow" data-focuszone-id="FocusZone27" tabindex="0" style="min-width: 1108.2px;">`

How to fix: 
Fix any of the following:
  Element has children which are not allowed (see related nodes)
  Element has no aria-busy="true" attribute

Environment: Microsoft Edge version 112.0.1722.48

## Expected Behavior
There should be a way to add aria-busy="true" for DetailsRow component

### Requested priority

High

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
